validation-test/ParseableInterface/verify_all_overlays.py

```diff
@@ -44,7 +44,7 @@
     else:
         continue
 
-    if module_name == "Swift" or module_name == "SwiftLang":
+    if module_name in ["Swift", "SwiftLang", "DifferentiationUnittest", "Python", "TensorFlow"]:
```

Comment:
Could you please make this change instead?
```
diff --git a/validation-test/ParseableInterface/verify_all_overlays.py b/validation-test/ParseableInterface/verify_all_overlays.py
index 6fb203fa15..778322b1c9 100755
--- a/validation-test/ParseableInterface/verify_all_overlays.py
+++ b/validation-test/ParseableInterface/verify_all_overlays.py
@@ -14,6 +14,9 @@

 # Expected failures by platform
 # -----------------------------
+# SWIFT_ENABLE_TENSORFLOW
+# FIXME: Re-enable for DifferentiationUnittest, Python, and TensorFlow modules.
+# macosx: DifferentiationUnittest
+# macosx: Python
+# macosx: TensorFlow
 # macosx: XCTest
 # ios: XCTest
 # tvos: XCTest
```

Reply:
This only seems sufficient for macosx?

Reply:
Ah, good point. Could you please add `SWIFT_ENABLE_TENSORFLOW` to the current solution instead?